stuff/cron/build_problem_rec_model.py

```diff
@@ -170,7 +172,7 @@ def save(self, output_path: str) -> None:
 
     def generate_weighted_pairs(self) -> pd.DataFrame:
         '''Assumes runs are sorted by submission time'''
-        tuples = []
+        tuples: Tuple[int, int, float] = []
```

Comment:
```suggestion
        tuples: List[Tuple[int, int, float]] = []
```

uno de estos días voy a agregar soporte de mypy para que nos diga los tipos correctos u_u